server/models/resource.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose");
+const organization = require("./organization");
+
+const resourceSchema = new mongoose.Schema({
+    title: {
+        required: true,
+        type: String
+    },
+    link: {
+        required: true,
+        type: String
+    },
+    ft_img: {
```

Comment:
Can you change this to `featured_img`? Just to be more consistent with `featured_img` in the Event type.